tools/ci_build/github/linux/docker/inference/x86_64/default/cpu/scripts/install_centos.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
All source code files should have copyright headers https://docs.opensource.microsoft.com/releasing/maintain/copyright-headers/